ontology-generator/ontology/fta-fmea-model.ttl

```diff
@@ -335,14 +347,26 @@ fta-fmea:triggers rdf:type owl:ObjectProperty ;
 
 
 #################################################################
-#    Data properties
+#    Data properties
 #################################################################
-
+
 ###  http://onto.fel.cvut.cz/ontologies/fta-fmea-application/from
 fta-fmea:from rdf:type owl:DatatypeProperty ;
               rdfs:domain fta-fmea:failure-rate-requirement .
 
 
+###  http://onto.fel.cvut.cz/ontologies/fta-fmea-application/has-ata-code
+fta-fmea:has-ata-code rdf:type owl:DatatypeProperty .
```

Comment:
We need to discuss naming conventions and how hard it would be to refactor everything with those conventions. It won't be most-likely easier than now.

Conventions from CSAT:
- entities contain only small letters, with words separated by "-" (even classes)
- object properties should start with :has-
- datatype properties typically do not start with :has-

Reply:
I do not see a problem of refactoring. The only problem I see is that existing data won't be compatible and would need to be refactored as well. 

However, I suggest to do this in a separate ticket.

Reply:
I suggest to do this in a separate ticket.
The only problem I can see is that we will have to also refactor old data, for example data in `fta-fmea-demo`.

I have some reservation for the proposed naming conventions:
- object properties should start with 'has' - in my opinion some properties do not need it. For example, `:activatedBy` should be refactored as `:has-activated-by`. I would prefer to refactor it as `:activated-by`.

Reply:
```
activates           
is-activated-by         
has-activating-behaviour
```